simulation_parameters/Constants.cs

```diff
@@ -109,6 +109,8 @@ public static class Constants
 
     public const float OXYTOXY_DAMAGE = 10.0f;
 
+    public const float PROJECTILE_DESPAWN_DELAY = 250;
```

Comment:
Why is the despawn delay 250 seconds? It should be about 3-5 seconds as I think that's the max time it will take for the particles to dissipate.

Reply:
adopted from the old PR

Reply:
Well it should be changed.